packages/shylu/shylu_dd/bddc/src/ProblemMakerBDDC.h

```diff
@@ -149,86 +199,31 @@ template <class LO, class GO, class SX, class SM> class ProblemMaker
     return &m_nodeGIDs[0];
   }
 
-  const LO* getRowBegin() const {
-    return &m_rowBegin[0];
-  }
-
-  const LO* getColumns() const {
-    return &m_columns[0];
-  }
-
-  const SX* getValues() const {
-    return &m_values[0];
-  }
-
   const SX* getLoadVector() const {
     return &m_loadVector[0];
```

Comment:
Use `std::vector::data()`; it's safe (with respect to bounds-checking STL implementations) in case the vector is empty.
```c++
return m_loadVector.data ();
```